doc/ModuleStartupOrder.md

```diff
@@ -116,7 +116,9 @@ The following sample deployment manifest illustrates how startupOrder values of
 
 In the sample deployment manifest shown above:
 
-* The `$edgeHub` module has been assigned a `startupOrder` value of 0.
+* The '$edgeAgent' schemaVersion has been set to 1.1 (or later).
+* The 'edgeAgent' module's image is 1.0.10 (or later).
```

Comment:
Per comment above, there is no need to update this image version.

Reply:
Ok, will remove.